CompareGraphs.py

```diff
@@ -44,9 +44,12 @@
         if isinstance(inObj, TH1):
             inObj.SetDirectory(0)
             inObj.Rebin(inputCfg['rebin'])
+        
+        if inputCfg['normalize']:
+            inObj.Scale(1./inObj.Integral())
+        if inputCfg['normalizecf']:
+            inObj.Scale(inputCfg['normalizecf'])
```

Comment:
Keep this block indented because `Scale` only works for `TH1`
```suggestion
        
            if inputCfg['normalize']:
                inObj.Scale(1./inObj.Integral())
            if inputCfg['normalizecf']:
                inObj.Scale(inputCfg['normalizecf'])
```